src/Mynt.Backtester/JsonCandleProvider.cs

```diff
@@ -22,14 +22,14 @@ public JsonCandleProvider(string folder)
         public List<Candle> GetCandles(string symbol, int period)
         {
             var basePath = AppDomain.CurrentDomain.BaseDirectory;
-            var filePath = Path.Combine(basePath, $"{folder}/{symbol}.db");
+            var filePath = Path.Combine(basePath, $"{folder}/{BacktestOptions.Exchange.ToLower()}_{symbol}.db");

```

Comment:
Now that this doesn't really do anything with JSON data anymore, perhaps this class needs to be renamed? :)